src/main/java/io/github/productboardlabs/kafka/serializers/AbstractJacksonKafkaAvroSerializer.java

```diff
@@ -69,7 +69,10 @@ protected int getSchemaId(SchemaMetadata schema) {
                 return schemaRegistry.getId(schema.getSubject(), new io.confluent.kafka.schemaregistry.avro.AvroSchema(schema.getSchema()));
             }
         } catch (Exception e) {
-            throw new SerializationException("Can not fetch schema", e);
+            // The exact schema - including doc comments - must be present in the registry otherwise schema mismatch will be reported.
+            // schema.getSubject() is e.g. "$topic.$eventName", schema.getSchema() is Avro schema in service resources/JAR,
+            // both has to exact-match a schema in Schema registry.
```

Comment:
I'd add a hint to check pb-kafka next to the "schema registry"

Reply:
This is a public repo, I intentionally added no internal info. But I can, no problem, let me know.